AzureCore/Source/Pipeline/Transport/HttpRequest.swift

```diff
@@ -42,6 +42,11 @@ public class HttpRequest {
         self.data = data
     }
 
+    public func text(encoding: String.Encoding = .utf8) -> String? {
```

Comment:
Would it make sense to have HttpRequest and HttpResponse both implement some kind of HttpCommon protocol that has this method as an extension? In that way, we'd avoid potential double maintenance.

Reply:
Fixed in e427da2